src/corelibs/U2Gui/src/ObjectViewModel.cpp

```diff
@@ -681,4 +681,17 @@ void GObjectViewAction::addToMenuWithOrder(QMenu *menu) {
     menu->addAction(this);
 }
 
+GObjectViewObjectHandler::~GObjectViewObjectHandler() {
+}
+
+bool GObjectViewObjectHandler::canHandle(GObjectView *, GObject *) {
```

Comment:
Check if this function is used somewhere. It yes - leave it as now, if not - make pure virtual

Reply:
It is a core function for the Handler class. Of course it is used.